src/databases/mongodb/models/rooms.model.js

```diff
@@ -29,22 +29,80 @@ const RoomSchema = new Schema({
 
 const room = mongoose.model("rooms", RoomSchema);
 
-const getRooms = function (db, dbName) {
-    return new Promise((resolve, reject) => {
-        try {
-            const roomsCollection = db.db(dbName).collection('rooms')
-            const document = roomsCollection.find().toArray(function (err, result) {
-                if (err) reject(err)
-
-                resolve(result)
-            })
-        } catch (err) {
-            reject(err)
-        }
-    })
+const getRooms = function (db, fechaActual) {
+	return new Promise((resolve, reject) => {
+    
+		try {
+      
+			const roomCollection = db.db('studybuddies').collection('rooms');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7180713)
```suggestion
			const roomCollection = db.db("studybuddies").collection('rooms');
```